Feature/cax portal@redux store alternative

vsamarora opened the pull request:
CPLP 374 Front end State Management Framework: Switch Mobx to Redux changes

fatihayyildiz committed:
Some basic approach to redux implementation
user invite list page connect to store
no async axios call yet implemented

vsamarora committed:
hooks update for registration and translation

vsamarora committed:
translation and withrouter changes

vsamarora committed:
conveted all files to function component

vsamarora committed:
current step to add in reducer

vsamarora committed:
API integration

vsamarora committed:
code clean up

vsamarora committed:
Merge branch 'feature/CAX-portal' of https://github.com/tractusx-team-portal-onboarding/tractusx into feature/CAX-Portal@redux-store-alternative

vsamarora committed:
user logout changes

vsamarora committed:
removind registrationoneid as it is not required

vsamarora committed:
removed duplicate route

vsamarora committed:
added sent invite and stepper

oyo commented on portal/code/company-portal/src/components/cax-header.tsx:
```diff
@@ -14,50 +14,41 @@
-        console.log(token);
-        UserService.logOut();
+    const logoutClick = () => {
+        // const token = UserService.getCachedToken();
```
maybe remove those comments - actually it becomes a useless wrapper function. see my comment below

oyo commented on portal/code/company-portal/src/components/cax-header.tsx:
```diff
@@ -71,16 +62,16 @@ class Header extends React.Component<IProp> {
-                                <div>({this.userRoles.join(", ")})</div>
-                                <div className='logout' onClick={() => this.logoutClick()}>Logout</div>
+                                <div>({userRoles.join(", ")})</div>
+                                <div className='logout' onClick={() => logoutClick()}>Logout</div>
```
```suggestion
                                <div className='logout' onClick={() => UserService.doLogout()}>Logout</div>
```

oyo replied:
although "Logout" is probably understood by everyone - shouldn't this string be i18n

vsamarora replied:
updated the changes

oyo reviewed (commented):
just some small remarks from my side. nothing critical though

vsamarora committed:
added logout onclick

fatihayyildiz committed:
User responsibilities page roles bug fixed
WIP for role error handling
email verify step is done
some code alignment and refactoring

vsamarora committed:
changes for connecting all pages with store

vsamarora committed:
added new component for verify step

fatihayyildiz reviewed (approved):
Looks alright at my side
